telosnetwork/teloscan
# #205 | Native wallets for advanced users

# Fixes[ #205](https://github.com/telosnetwork/teloscan/issues/250)
## Description

This PR tweaks the login modal to make it clearer to users that Native wallets (i.e. anything other than metamask) are for advanced users / specific use cases.

- tweaks layout of login modal
- increases Wombat logo resolution
- adds warning for native wallet view

## Test scenarios
- go to the deploy preview, https://deploy-preview-264--benevolent-tapioca-f74277.netlify.app/
- click `CONNECT WALLET` in the top right corner (you will need to log out first if you are logged in)
    - the modal should have tabs "EVM Wallets" and "Advanced"
    - layout of the metamask logo should look good on all screen sizes
- click `Advanced` tab
    - layout of logos should look good on various screen sizes
    - wombat logo should not be pixelated (compare to the same view on https://teloscan.io)
    - there should be a warning paragraph above the icons



## Checklist:
